myapp/spec/system/tasks_spec.rb

```diff
@@ -0,0 +1,141 @@
+require 'rails_helper'
+describe 'タスク管理機能', type: :system do
+  # タスクを作成
+  user = User.create!(name: 'name1')
+  let!(:task_a) { FactoryBot.create(:task, title: '最初のタスク', description: '最初のタスクを実施する', status: '1', label: '1', user_id: user[:id]) }
+  let!(:task_b) { FactoryBot.create(:task, title: '２つ目のタスク', description: '２つ目のタスクを実施する', status: '1', label: '2', user_id: user[:id]) }
+  # タスクが表示される期待動作を共通化
+  shared_examples_for 'タスクが表示される' do
+    it { expect(page).to have_content '最初のタスク' }
+  end
+  shared_examples_for '２つ目のタスクが表示される' do
+    it { expect(page).to have_content '２つ目のタスク' }
+  end
+  describe '一覧表示機能' do
+    context 'タスクが1件存在する場合' do
+      before do
+        # 確認するパス（URL）を設定
+        visit tasks_path
+      end
+      it_behaves_like 'タスクが表示される'
+    end
+    context 'タスクが2件(複数)存在する場合' do
+      before do
+        visit tasks_path
+      end
+      it_behaves_like 'タスクが表示される'
+      it_behaves_like '２つ目のタスクが表示される'
+    end
+  end
```

Comment:
ブロック区切りや処理のまとまりなど、適切に空行を挿入して区切りをわかりやすくましょう。
例としてここだけ抜粋しましたが、全体的に見直していただけたらと思います。
```suggestion
  shared_examples_for 'タスクが表示される' do
    it { expect(page).to have_content '最初のタスク' }
  end
  
  shared_examples_for '２つ目のタスクが表示される' do
    it { expect(page).to have_content '２つ目のタスク' }
  end
  
  describe '一覧表示機能' do
    context 'タスクが1件存在する場合' do
      before do
        # 確認するパス（URL）を設定
        visit tasks_path
      end
      
      it_behaves_like 'タスクが表示される'
    end
    
    context 'タスクが2件(複数)存在する場合' do
      before do
        visit tasks_path
      end
      
      it_behaves_like 'タスクが表示される'
      it_behaves_like '２つ目のタスクが表示される'
    end
  end
```

Reply:
空行がなく、見づらいspecになっていたため、改行し、わかりやすくしました。